website/docs/developer/overview.md

```diff
@@ -0,0 +1,16 @@
+---
+id: overview
+title: Overview
+sidebar_label: Overview
+---
+
+OpenAttestation comes with a set of tools that help you to build, verify or share documents depending on your need.
+
+In this section you will discover all the tools that OpenAttestation offers and how to use them:
+
+- [Open Attestation](/docs/developer/open-attestation) is the core framework that allows you to create OA documents such as, transferable record or verifiable document.
+- [Open Attestation (Verify)](/docs/developer/open-attestation-verify) helps you to verify that documents are valid OA documents.
+- [Open Attestation (Encryption)](/docs/developer/open-attestation-encryption) assists you to share and store your OA documents in a secure way.
+- [Decentralized Renderer](/docs/developer/decentralized-renderer) allows recipient of OA documents to view their document in a human readable format.
+
```

Comment:
wait now i've realise this is the same content as `Components` overview, can we write something different here, like explaining the need/overview for this `Developer` section